pipdeptree.py

```diff
@@ -851,22 +873,35 @@ def main():
 
     show_only = set(args.packages.split(',')) if args.packages else None
     exclude = set(args.exclude.split(',')) if args.exclude else None
+    include_re = None
+    exclude_re = re.compile(args.exclude_regex) if args.exclude_regex else None
 
-    if show_only is not None or exclude is not None:
-        tree = tree.filter(show_only, exclude)
-
-    if args.json:
-        print(render_json(tree, indent=4))
-    elif args.json_tree:
-        print(render_json_tree(tree, indent=4))
-    elif args.output_format:
-        output = dump_graphviz(tree,
-                               output_format=args.output_format,
-                               is_reverse=args.reverse)
-        print_graphviz(output)
-    else:
-        render_text(tree, args.all, args.freeze)
+    if show_only is not None or exclude is not None or exclude_re is not None:
+        tree = tree.filter(show_only, exclude, include_re, exclude_re)
 
+    output = sys.stdout
+    try:
+        if args.output:
+            output = TextIOWrapper(open(args.output, 'wb'))
```

Comment:
Can you please confirm if this works in python 2.7? Don't want to remove support for python 2.7 yet. 

I tried the following on python 2.7.14 and got an error

```
In [8]: print('abc', file=TextIOWrapper(open('/tmp/hello.txt', 'wb')))
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-8-e0a5198379f0> in <module>()
----> 1 print('abc', file=TextIOWrapper(open('/tmp/hello.txt', 'wb')))

AttributeError: 'file' object has no attribute 'readable'
```